internal/services/manageddevopspools/managed_devops_pool_resource.go

```diff
@@ -0,0 +1,751 @@
+
+		"location": commonschema.Location(),
+
+		"azure_devops_organization_profile": {
+			Type:     pluginsdk.TypeList,
+			Required: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"organization": {
+						Type:     pluginsdk.TypeList,
+						Required: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"url": {
+									Type:     pluginsdk.TypeString,
+									Required: true,
+									ValidateFunc: validation.All(
+										validation.IsURLWithHTTPS,
+										validation.StringMatch(
+											regexp.MustCompile(`[a-zA-Z0-9]$`),
+											"url must end with a letter or number",
+										),
+									),
+								},
+
+								"parallelism": {
+									Type:         pluginsdk.TypeInt,
+									Optional:     true,
+									ValidateFunc: validation.IntBetween(1, 10000),
+								},
+
+								"projects": {
+									Type:     pluginsdk.TypeList,
+									Optional: true,
+									Elem: &pluginsdk.Schema{
+										Type:         pluginsdk.TypeString,
+										ValidateFunc: validate.ProjectName,
+									},
+								},
+							},
+						},
+					},
+
+					"permission_profile": {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"kind": {
+									Type:         pluginsdk.TypeString,
+									Required:     true,
+									ValidateFunc: validation.StringInSlice(pools.PossibleValuesForAzureDevOpsPermissionType(), false),
+								},
+
+								"administrator_account": {
+									Type:     pluginsdk.TypeList,
+									Optional: true,
+									MaxItems: 1,
+									Elem: &pluginsdk.Resource{
+										Schema: map[string]*pluginsdk.Schema{
+											"groups": {
+												Type:     pluginsdk.TypeList,
+												Optional: true,
+												Elem: &pluginsdk.Schema{
+													Type:         pluginsdk.TypeString,
+													ValidateFunc: validate.Email,
+												},
+												AtLeastOneOf: []string{"azure_devops_organization_profile.0.permission_profile.0.administrator_account.0.groups", "azure_devops_organization_profile.0.permission_profile.0.administrator_account.0.users"},
+											},
+
+											"users": {
+												Type:     pluginsdk.TypeList,
+												Optional: true,
+												Elem: &pluginsdk.Schema{
+													Type:         pluginsdk.TypeString,
+													ValidateFunc: validate.Email,
+												},
+												AtLeastOneOf: []string{"azure_devops_organization_profile.0.permission_profile.0.administrator_account.0.groups", "azure_devops_organization_profile.0.permission_profile.0.administrator_account.0.users"},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+
+		"dev_center_project_resource_id": commonschema.ResourceIDReferenceRequired(&projects.ProjectId{}),
```

Comment:
When referencing an Azure Resource ID we generally omit `resource` from the field name

```suggestion
		"dev_center_project_id": commonschema.ResourceIDReferenceRequired(&projects.ProjectId{}),
```